.github/workflows/release_js_sdk.yml

```diff
@@ -1,13 +1,16 @@
 name: Release JS SDK
 
 on:
-  push:
-    # Pattern matched against refs/tags
-    # This workflow will run when a tag starting with 'js-' is pushed
-    # For example, 'js-v1.0.0' or 'js-release-2.3.1'
-    tags:        
-      - 'js-*'  
-    
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to publish (e.g., 1.0.0)'
+        required: true
+        type: string
```

Comment:
Consider adding pattern validation for the version input to ensure it follows semver format:
```yaml
version:
  description: 'Version to publish (e.g., 1.0.0)'
  required: true
  type: string
  pattern: '^[0-9]+\.[0-9]+\.[0-9]+(-rc\.[0-9]+)?$'
```

Reply:
@himanshu-dixit I believe, it's good to add a pattern check. what do you think?

Reply:
AI hallucination, pattern is not supported
<img width="1238" alt="Screenshot 2024-11-29 at 1 18 43 PM" src="https://github.com/user-attachments/assets/039fc1ac-06ee-40e1-8f16-d91381b09c0b">

Reply:
lol my bad. LGTM